pylhc/machine_settings_info.py

```diff
@@ -17,28 +17,32 @@
 
 .. code-block:: none
 
-    usage: machine_settings_info.py [-h] [--time TIME] [--start_time START_TIME]
-                                    [--knobs KNOBS [KNOBS ...]] [--accel ACCEL]
-                                    [--output_dir OUTPUT_DIR] [--knob_definitions]
-                                    [--source SOURCE] [--log]
-
-    optional arguments:
-    -h, --help            show this help message and exit
-    --time TIME           UTC Time as 'Y-m-d H:M:S.f' format or ISO.
-                          Acts as point in time or end time
-                         (if ``start_time`` is given).
-    --start_time START_TIME
-                            UTC Time as 'Y-m-d H:M:S.f' format.
-                            Defines the beginning of the time-range.
-    --knobs KNOBS [KNOBS ...]
-                            List of knobnames.
-    --accel ACCEL         Accelerator name.
-    --output_dir OUTPUT_DIR
-                            Output directory.
-    --knob_definitions    Set to extract knob definitions.
-    --source SOURCE       Source to extract data from.
-    --log                 Write summary into log
-                          (automatically done if no output path is given).
+   usage: machine_settings_info.py [-h] [--time TIME] [--start_time START_TIME]
+                                   [--knobs KNOBS [KNOBS ...]] [--accel ACCEL]
+                                   [--beamprocess BEAMPROCESS] [--output_dir OUTPUT_DIR]
+                                   [--knob_definitions] [--source SOURCE] [--log]
+
+  optional arguments:
+  -h, --help            show this help message and exit
+  --time TIME           UTC Time as 'Y-m-d H:M:S.f' or ISO format or AccDatetime object.
+                        Acts as point in time or end time (if ``start_time`` is given).
+  --start_time START_TIME
+                        UTC Time as 'Y-m-d H:M:S.f' or ISO format or AccDatetime object.
+                        Defines the beginning of the time-range.
+  --knobs KNOBS [KNOBS ...]
+                        List of knobnames. If `None` (or omitted) no knobs will be extracted.
+                        If it is just the string ``'all'``, all knobs will be extracted
+                        (can be slow). Use the string ``'default'`` the main knobs
```

Comment:
Not updated here, but that's ok

Reply:
fuck. I thought I checked there and thought it used a shorter description. No idea